docs/topics/proc_updating-booster-deployment-configuration.adoc

```diff
@@ -0,0 +1,72 @@
+[[proc_updating-booster-deployment-configuration]]
+= Updating the Deployment Configuration of a Booster
+
+The deployment configuration for a booster contains information related to deploying and running the booster in OpenShift, such as route information or readiness probe location. 
+The deployment configuration of a booster is stored in a set YAML files. 
+For boosters that use the Fabric8 Maven Plugin, the YAML files are located in the `src/main/fabric8/` folder. For {NodeJS} boosters using Nodeshift, the YAML files are located in the `.nodeshift` folder.
```

Comment:
I don't know about Nodeshift, but there's a very important point when it comes to Fabric8 Maven plugin: the YAML files stored in `src/main/fabric8` can, but _don't have to_ be the full OpenShift resource definitions. In case of the boosters, they often _aren't_! FMP is able to fill in the missing parts. Which can be convenient, but also confusing and even misleading (for example, if you specify a `Deployment` but don't add the _full_ specification, FMP actually changes that to a `DeploymentConfig` when it knows it's targetting OpenShift!).

Not sure how to express that in docs, but it's pretty important.

Reply:
@Ladicek Would you know if this opinionated feature of FMP is mentioned anywhere in the FMP docs? I'll take a look and if I find any such information documented there, I'd propose we link to that explanation. On the other hand, if there is no mention of it anywhere, I'd be happy to write something up to add that information, since it is IMHO important.